test/ipld-bitcoin.js

```diff
@@ -83,16 +83,12 @@ module.exports = (repo) => {
         }
       ], store)
 
-      function store () {
-        pull(
-          pull.values([
-            { node: node1, cid: cid1 },
-            { node: node2, cid: cid2 },
-            { node: node3, cid: cid3 }
-          ]),
-          pull.asyncMap((nac, cb) => resolver.put(nac.node, { cid: nac.cid }, cb)),
-          pull.onEnd(done)
-        )
+      async function store () {
+        const nodes = [node1, node2, node3]
+        const result = resolver.put(nodes, multicodec.BITCOIN_BLOCK)
+        ;[cid1, cid2, cid3] = await result.all()
```

Comment:
The leading semicolon makes me sad. I know why it's necessary, but it still makes me sad.

What about changing the variable scope and keeping everything contained within the function so you don't need it?  Yum yum, side-effect free.

Reply:
The tests really need a revisit at one point. It's a lot of duplicated code. I wanted to keep the changes as minimal as possible for this PR.

Reply:
Ha!